naplib/naplab/process_ieeg.py

```diff
@@ -65,7 +66,12 @@ def process_ieeg(
         as those within ``stim_dir``. E.g. {'aud': './stimuli', 'aud_spk1': './stimuli_spk1', 'aud_spk2': './stimuli_spk2'}
     data_type : str, default='infer'
         One of {'edf', 'tdt', 'nwb', 'pkl', 'infer'}. The data type of the raw neural data to load.
-    rereference_grid : Optional[np.ndarray], default=None
+    elec_names: Optional[Union[str, Sequence[str]]] path-like or sequence of strings, default=None
+        Electrode labels for all data channels read from ``data_path``. Should either be the path to a text file where
+        each line is the label of an electrode contact, or a list of strings where each element is the label of an electrode
+        contact. In both cases, the number of labels provided should match the number of data channels in ``data_path``. If
+        None, the labels included in the data file will be used.
+    rereference_grid : Optional[Union[np.ndarray, str]], default=None
         If not None, then data are re-referenced based on this referencing scheme. If a numpy array, then
```

Comment:
update this description to include the option for "subject" to be given as a string